apm-sniffer/optional-plugins/logger-plugin/src/test/java/org/apache/skywalking/apm/plugin/logger/ContextConfigDefaultTest.java

```diff
@@ -24,7 +25,7 @@
 public class ContextConfigDefaultTest {
     @Test
     public void testDefaultConfig() {
-        ContextConfig config = ContextConfig.getInstance();
+        ContextConfig config = ContextConfig.getLatestConfig();
```

Comment:
@zifeihan Why we should write another method named `getLastsConfig()` for UT. It seems that when any UT starts, `initContextConfig()` will be executed. Just curious :smile:

Reply:
@vcjmhg Hello, because maven test is in a jvm instance, the loading configuration file content is in singleton mode, so if the file content is modified in the test case, it will not take effect.

Reply:
@vcjmhg you can test success for every UT by manual, but it will failed in maven automatic test.

Reply:
ok , I got it. Thank u!